.travis.yml

```diff
@@ -1,51 +1,9 @@
-dist: trusty
-sudo: required
 language: java
-os:
-  - linux
-
-env:
-  - V=0.4.5
-
-before_install:
```

Comment:
We plan use `bazel` in this project. That is why we change `.travis.yml` at first place. After changes, travis becomes much slower. When we use `mvn`, it only takes one or two minute. But `bazel`needs 4 minutes. The reason could be using `bazel` requires `sudo`.